docs/release-notes/cli-user-edit.rst

```diff
@@ -0,0 +1,7 @@
+:orphan:
+
+**New Features**
+
+-  CLI: Add a new CLI command ``det user edit <target_user> [--display-name] [--remote] [--active]
+   [--admin] [--username]`` that allows the user to edit mutliple fields for the target user. Old
+   methods for editing users will still be available, but deprecated.
```

Comment:
**New Features**

-  CLI: Add a new CLI command ``det user edit <target_user> [--display-name] [--remote] [--active]
   [--admin] [--username]`` that allows the user to edit multiple fields for the target user. Old
   methods for editing users will still be available, but deprecated.